.github/workflows/lint-test.yaml

```diff
@@ -11,6 +11,12 @@ on:
 
 env:
   HAS_BUILDPULSE_SECRETS: ${{ secrets.BUILDPULSE_ACCESS_KEY_ID != '' && secrets.BUILDPULSE_SECRET_ACCESS_KEY != '' }}
+  CARGO_INCREMENTAL: "0"
```

Comment:
slightly faster builds and smaller cache sizes when disabling incremental compilation as described here https://matklad.github.io/2021/09/04/fast-rust-builds.html